stormpath/stormpath-sdk-angularjs
# Export name bug

Fixes #80.

This PR makes it possible to include the SDK using browserify or webpack.
- `stormpath-sdk-angularjs.js` exports `'stormpath'` instead of `'ui.router'`
- `stormpath-sdk-angularjs.tpls.js` exports `'stormpath.templates`
### How to verify
1. Checkout this branch
2. Copy the `dist/` folder into your AngularJS project (you can use [this boilerplate](https://github.com/basti1302/angular-browserify))
3. Modify your `package.json` to include these lines:
   
   ``` json
   "browser": {
     "stormpath": "./lib/stormpath-sdk-angularjs.js",
     "stormpath.templates": "./lib/stormpath-sdk-angularjs.tpls.js"
   }
   ```
4. In `app/js/app.js`, require `stormpath` and `stormpath.templates` in dependencies:
   
   ``` javascript
   var app = angular.module('todoApp', [
     require('angular-route'),
     require('stormpath'),
     require('stormpath.templates')
   ]);
   ```
5. Make sure that the templates still work by using the login directive in `app/views/todos.html`:
   
   ``` html
   ...
   <div sp-login-form></div>
   ...
   ```
6. Build the app:
   
   ``` bash
   $ gulp
   ```
7. Serve the app using the [`SPA Development Server`](https://github.com/stormpath/stormpath-spa-dev-server):
   
   ``` bash
   $ stormpath-spa-dev-server <path-to-your-app>/app/dist
   ```
8. Verify that the form gets rendered and that the SDK is included properly
9. To verify all functionality, please exchange the `ngRouter` for `UI Router` instead
